cmd/agent/common/autoconfig_nodocker.go

```diff
@@ -8,14 +8,40 @@
 package common
 
 import (
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/collector"
+	"github.com/DataDog/datadog-agent/pkg/collector/autodiscovery"
+	"github.com/DataDog/datadog-agent/pkg/collector/loaders"
+	"github.com/DataDog/datadog-agent/pkg/collector/providers"
 	log "github.com/cihub/seelog"
 )
 
-// SetupAutoConfig placeholder if docker is disabled
+// SetupAutoConfig only starts the Autoconfig subsystem if Docker is not available
 func SetupAutoConfig(confdPath string) {
-	log.Debugf("AutoDiscovery is only supported with docker, disabling")
+	// create the Collector instance and start all the components
+	// NOTICE: this will also setup the Python environment
+	Coll = collector.NewCollector("")
```

Comment:
any reason not to set any python paths by default when docker is disabled?

(I understand it's not an issue for the puppy agent since it's a not shipping the python part anyway, but this file is supposed to be related to docker only :) )

Reply:
Good catch, this is an oversight from the first version of the fix, before removing `docker` from the windows build